### ISSUE TITLE
Don't align markdown titles to the center, but to the left

### ISSUE THREAD

**Issue by oryband** [TheR1D/shell_gpt#581]
See screenshot. It just makes things more difficult to read. Any chance to align them properly?

![image](https://github.com/TheR1D/shell_gpt/assets/115387/55e4493a-2af3-4492-bc36-d2e8a2577ca7)

**Comment by will-wright-eng** [comment]
as to your second question, it looks like the logic for handling all types of stdin is tricky ([source](https://github.com/TheR1D/shell_gpt/blob/7678afe46f46fcbaa97b810942b1dfae71111212/sgpt/app.py#L163-L170))

**Comment by Olen** [comment]
I agree that the centered headings are hard to read. However, this seems to be built into the "rich" library:
https://rich.readthedocs.io/en/latest/_modules/rich/markdown.html
(search for "Heading" and look at "align")

It can be overridden, but not by a simple config change. One would need to override the default behaviour, which is prone to error and unintended consequences...

There is actually a PR to Rich to fix this: https://github.com/Textualize/rich/pull/3830
